src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -24,4 +26,14 @@ public static boolean isEven(int a) {
     public static boolean isOdd(int a) {
         return !isEven(a);
     }
+
+    public static int countOdd(int[] array) {
+        checkNotNull(array);
+        int count = 0;
+        for (int j : array) {
+            if (isOdd(j))
```

Comment:
It's a common practice to use braces with if, for and other statements. As an example see "[Google Java Style Guide: 4.1.1 Braces are used where optional](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)".

See also the reason why we have to do it this way: https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces 
Basically it's just an error-prone thing to not use curly braces :)

So please put open and closing braces after the if statement